bin/manuscripts

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
-#
+####
```

Comment:
Why the extra #?

Reply:
the git commit command wasnt showing that changes were made in this file, so i added these exrra #